extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -153,7 +153,7 @@ export const ResetButton: string = localize('reset', "Reset");
 export const OptionDescription: string = localize('optionDescription', "Option Description");
 export const OptionName: string = localize('optionName', "Option Name");
 export const OptionInclude: string = localize('Include', "Include");
-
+export const IncludeCompositeObjects: string = localize('IncludeCompositeObjectsOption', 'Include composite objects');
```

Comment:
is this string also localized when coming from DacFx? Otherwise this won't match what's in the optionsMapTable if the user is using a non-English language

Reply:
I have this in my notes for testing localization...will prioritize it!!

Since the options are coming from the DacFx which are localized, there shouldn't be any issue with localized strings! i'll make sure to test it with help! Thanks for raising this question!